src/entrypoints/web/tsconfig.json

```diff
@@ -0,0 +1,48 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "es6"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "bundler",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "incremental": true,
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ],
+    "baseUrl": ".",
+    "paths": {
+      "@/*": [
+        "./*"
+      ],
+      "@/core/*": [
+        "../../core/*"
+      ],
+      "@/providers/*": [
+        "../../providers/*"
+      ]
```

Comment:
We are declaring these aliases but we are not using this consistently across all file imports. We need to refactor the other sourcecode files of the web folder to use these alias